Tests/SwiftFormatPrettyPrintTests/SequenceExprFoldingTests.swift

```diff
@@ -0,0 +1,204 @@
+import SwiftFormatPrettyPrint
+import SwiftSyntax
+import XCTest
+
+final class SequenceExprFoldingTests: XCTestCase {
```

Comment:
Could you also add a test demonstrating and testing how to add a custom operator programmatically and have it folded properly?

Reply:
Done.